assets/static/js/common/dise_api.js

```diff
@@ -1,6 +1,6 @@
 (function() {
-    var base = 'https://dise.klp.org.in/api/';
-    var DEFAULT_ACADEMIC_YEAR = '13-14';
+    var base = 'http://dise.odisha.ilp.org.in/api/';
+    var DEFAULT_ACADEMIC_YEAR = '15-16';
```

Comment:
Change these values based on what and where you're hosting DISE (`dise_dashboard`)